documentation/working-with-testcafe-studio/recording-tests/test-actions/assertions.md

```diff
@@ -0,0 +1,195 @@
+---
+# Assertions
+
+The **Assertions** category contains actions that perform different verifications.
+
+* ![Action icon](../../../../images/actions/assertion-eql-icon.svg) [Equals](#equals)
+* ![Action icon](../../../../images/actions/assertion-not-eql-icon.svg) [Not Equals](#not-equals)
+* ![Action icon](../../../../images/actions/assertion-ok-icon.svg) [Ok](#ok)
+* ![Action icon](../../../../images/actions/assertion-not-ok-icon.svg) [Not Ok](#not-ok)
+* ![Action icon](../../../../images/actions/assertion-contains-icon.svg) [Contains](#contains)
+* ![Action icon](../../../../images/actions/assertion-not-contains-icon.svg) [Not Contains](#not-contains)
+* ![Action icon](../../../../images/actions/assertion-type-of-icon.svg) [Type of](#type-of)
+* ![Action icon](../../../../images/actions/assertion-not-type-of-icon.svg) [Not Type of](#not-type-of)
+* ![Action icon](../../../../images/actions/assertion-gt-icon.svg) [Greater than](#greater-than)
+* ![Action icon](../../../../images/actions/assertion-gte-icon.svg) [Greater than or Equals to](#greater-than-or-equals-to)
+* ![Action icon](../../../../images/actions/assertion-lt-icon.svg) [Less than](#less-than)
+* ![Action icon](../../../../images/actions/assertion-lte-icon.svg) [Less than or Equals to](#less-than-or-equals-to)
+* ![Action icon](../../../../images/actions/assertion-within-icon.svg) [Within](#within)
+* ![Action icon](../../../../images/actions/assertion-not-within-icon.svg) [Not Within](#not-within)
+* ![Action icon](../../../../images/actions/assertion-match-icon.svg) [Match](#match)
+* ![Action icon](../../../../images/actions/assertion-not-match-icon.svg) [Not Match](#not-match)
+
+## Assertion Parameters and Options
+
+When you add an assertion to a test, you need to set up its parameters.
+
+Each assertion has an **Actual** parameter that takes a comparison value. This parameter can take any value. If you need to compare a property of a particular DOM node, you can pass a [selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/)'s property to this parameter. You can enter a selector's property to the **Actual** field or generate it during test recording by selecting a target element on the tested webpage.
+
+To generate a Selector's property, follow the steps:
+
+1. Click the ![Element Picker](../../../../images/getting-started/element-picker-icon.png) **Element Picker** button next to the **Actual** field.
+2. Select a target element on the tested webpage using an element picker. TestCafe Studio generates an element selector and adds it to the **Actual** field.
+3. Select a desired property from the list.
+
+Some of assertions have the **Expected** parameter that takes an expected value. You can enter a desired value or add the current property value by clicking the ![Copy the value to the Expected field](../../../../images/getting-started/copy-value-icon.png) button.
```

Comment:
I think we should say that assertions' editors contains code which we evaluate and show result in `value` editors.
Because if you want to write e.g.  string in typeText action you just type in `text` in text input. But if you want to write string in expected editor you should write `'text'`.